custom_components/luxtronik/sensor.py

```diff
@@ -394,16 +396,18 @@ def _calc_next_evu_event_minutes(self) -> int | None:
             return None
         evu_hours = (24 if evu_time < time_now else 0) + evu_time.hour
         weekday = datetime.today().weekday()
+        if not isinstance(self._attr_cache[SA.EVU_DAYS], list):
```

Comment:
here's a bug. it has the checked if the attribute is a string. if yes, convert it to a list to append a new weekday. Please replace with the following:

`
        if isinstance(self._attr_cache[SA.EVU_DAYS], str):
            self._attr_cache[SA.EVU_DAYS] = self._attr_cache[SA.EVU_DAYS].split(',')
        elif not isinstance(self._attr_cache[SA.EVU_DAYS], list):
            self._attr_cache[SA.EVU_DAYS] = list()		# failsafe
`